IonDotnet/IonType.cs

```diff
@@ -1,32 +1,57 @@
-﻿namespace IonDotnet
+﻿using System;
+
+namespace IonDotnet
 {
-    public enum IonType : short
+    /// <summary>
+    /// Defines the static constant Ion types.
+    /// </summary>
+    public class IonType : IonTypeEnumeration
     {
-        None = -1,
-        Null = 0,
-        Bool = 1,
-        Int = 2,
-        Float = 3,
-        Decimal = 4,
-        Timestamp = 5,
-        Symbol = 6,
-        String = 7,
-        Clob = 8,
-        Blob = 9,
-        List = 10,
-        Sexp = 11,
-        Struct = 12,
-        Datagram = 13
+        public static readonly IonType None = new IonType("none", -1);
```

Comment:
None is no longer needed. Instead, where-ever "None" is assigned now the entire field is just assigned as null.

Reply:
Done.